test/functional/test_drive_error_inject.py

```diff
@@ -0,0 +1,128 @@
+"""
+*********************************************************
+Copyright @ 2015 EMC Corporation All Rights Reserved
+*********************************************************
+"""
+import unittest
+import os
+import time
+import re
+import tempfile
+import subprocess
+import socket
+import json
+from infrasim import model
+from infrasim import helper
+from test import fixtures
+from infrasim.helper import UnixSocket
+
+old_path = os.environ.get('PATH')
+new_path = '{}/bin:{}'.format(os.environ.get('PYTHONPATH'), old_path)
+image = os.environ.get('TEST_IMAGE_PATH') or "/home/infrasim/jenkins/data/ubuntu14.04.4.qcow2"
+
+conf = {}
+error_inject_list = [
+    'nvme -i dev-nvme-0 -n 1 -s 4 -t 0',
+    'nvme -i dev-nvme-0 -n 1 -s 5 -t 0',
+    'nvme -i dev-nvme-0 -n 1 -s 6 -t 0',
+    'nvme -i dev-nvme-0 -n 1 -s 18 -t 0',
+    'nvme -i dev-nvme-0 -n 1 -s 25 -t 0',
+    'nvme -i dev-nvme-0 -n 1 -s 130 -t 0',
+    'nvme -i dev-nvme-0 -n 1 -s 132 -t 0',
+    'nvme -i dev-nvme-0 -n 1 -s 128 -t 2',
+    'nvme -i dev-nvme-0 -n 1 -s 129 -t 2',
+    'nvme -i dev-nvme-0 -n 1 -s 130 -t 2',
+    'nvme -i dev-nvme-0 -n 1 -s 131 -t 2',
+    'nvme -i dev-nvme-0 -n 1 -s 132 -t 2',
+    'nvme -i dev-nvme-0 -n 1 -s 133 -t 2',
+    'nvme -i dev-nvme-0 -n 1 -s 134 -t 2',
+    'nvme -i dev-nvme-0 -n 1 -s 135 -t 2'
+]
+
+
+def setup_module():
+    TOOL = "drive_error_injection_cli.py"
+    DOWNLOAD_URL = "https://raw.githubusercontent.com/InfraSIM/tools/master/error_injection/drive_error_injection_cli.py"
+    MD5_KCS_IMG = "8e0fd62143d6e9bdfa5c21026789a9f2"
+    try:
+        helper.fetch_image(DOWNLOAD_URL, MD5_KCS_IMG, TOOL)
+    except InfraSimError, e:
+        print e.value
+        assert False
+    os.environ['PATH'] = new_path
+
+
+def teardown_module():
+    os.environ['PATH'] = old_path
+
+
+class test_error_inject(unittest.TestCase):
+
+    @staticmethod
+    def start_node():
+        global conf
+        nvme_config = fixtures.NvmeConfig()
+        conf = nvme_config.get_node_info()
+        node = model.CNode(conf)
+        node.init()
+        node.precheck()
+        node.start()
+        time.sleep(10)
```

Comment:
"sleep(10)" is useless here. remove it.
helper.port_forward(node) will wait until qemu to start.